packages/twenty-server/src/engine-metadata/field-metadata/dtos/default-value.input.ts

```diff
@@ -16,6 +17,12 @@ export class FieldMetadataDefaultValueString {
   value: string | null;
 }
 
+export class FieldMetadataDefaultValueJson {
+  @ValidateIf((_object, value) => value !== null)
+  @IsJSON()
```

Comment:
I haven't tested extensively everything related to the JSON field type. It will be easier to test once we build the frontend in a followup ticket